src/infrastructure/RequestHelper.ts

```diff
@@ -56,13 +99,21 @@ export async function get(
       ...options,
     });
 
-    return [...body, ...more];
+    return [...(Array.isArray(body) ? body : []), ...(Array.isArray(more) ? more : [])];
   }
 
   return (query.page || maxPages) && showPagination ? { data: body, pagination } : body;
 }
 
-export function stream(service, endpoint: string, options: BaseRequestOptions = ({} = {})) {
+export function stream(
+  service: BaseService,
+  endpoint: string,
+  options: BaseRequestOptions = ({} = {}),
```

Comment:
What is this syntax for the default value?

Reply:
It was already there. I agree that is strange. The change is only due to prettier.